### ISSUE TITLE
Bug Crashing IR demon when using pocket edition connected to GPIO. 

### ISSUE THREAD

**Issue by SnipeXR** [polyvision/EasyRaceLapTimer#82]
Initial investigation in version 0.6 the ir daemon crashes when getting bombarded with trigger events from pocket edition. 
Current patch to keep ir daemon running is a bash script in a forever loop recovering the IR_Deamon App. 
Crashes seem intermittent and are more likely to occur when the trigger condition happens.

**Comment by polyvision** [comment]
Yeah, it crashes a lot at present with the pocket edition. Working on it!

**Comment by polyvision** [comment]
Just some news. I'm working on a NodeJS version of the ir_daemon. It's much less code and easier for other people to contribute.
